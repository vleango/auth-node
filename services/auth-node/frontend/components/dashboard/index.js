import React from 'react';

export const DashboardPage = () => (
  <div>
    <h1>Dashboard</h1>
    <p>Welcome</p>
  </div>
);

export default DashboardPage;
