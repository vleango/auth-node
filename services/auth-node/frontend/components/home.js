import React from 'react';

export const HomePage = () => (
  <div>
    <h1>App</h1>
  </div>
);

export default HomePage;
