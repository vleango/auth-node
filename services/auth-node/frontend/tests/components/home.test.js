import React from 'react';
import { shallow } from 'enzyme';
import { HomePage } from '../../components/home';

let wrapper = shallow(<HomePage />);

describe('Components', () => {
  describe('Home', () => {

    it('should correctly render HomePage', () => {
      expect(wrapper).toMatchSnapshot();
    });

  });
});
