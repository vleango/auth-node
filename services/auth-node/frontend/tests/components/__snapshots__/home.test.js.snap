// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components Home should correctly render HomePage 1`] = `
<div>
  <h1>
    App
  </h1>
</div>
`;
